### PR TITLE
pre-release v2.1.0-rc11 for testnet

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: config_testnet.yaml [hunk 1]
```diff
@@ -17,7 +17,6 @@ chain:
   candidateIndexDBPath: "/var/data/candidate.index.db"
   stakingIndexDBPath: "/var/data/staking.index.db"
   contractStakingIndexDBPath: "/var/data/contractstaking.index.db"
-  sgdIndexDBPath: "/var/data/sgd.index.db"
```
Comment (dustinxie) [inline config_testnet.yaml:20]: this feature deprecated

File: genesis_testnet.yaml [hunk 2]
```diff
@@ -6,6 +6,7 @@ account:
   replayDeployerWhitelist:
   - 0x3fab184622dc19b6109349b94811493bf2a45362
   - 0x4c8d290a1b368ac4728d83a9e8321fc3af2b39b1
+  - 0xa71ec2ea4a0df8ba132dfe15bb76f6408a2f887f
```
Comment (dustinxie) [inline genesis_testnet.yaml:9]: this is address for one of polygon's deterministic deployer, requested by Leo

File: genesis_testnet.yaml [hunk 3]
```diff
@@ -33,6 +34,7 @@ blockchain:
   sumatraHeight: 24527521
   tsunamiHeight: 25226641
   upernavikHeight: 26984161
+  vanuatuHeight: 28744561
```
Comment (dustinxie) [inline genesis_testnet.yaml:37]: placeholder time Tuesday 9:30am, we can determine the specific time later

File: genesis_testnet.yaml [hunk 4]
```diff
@@ -33,6 +34,7 @@ blockchain:
   sumatraHeight: 24527521
   tsunamiHeight: 25226641
   upernavikHeight: 26984161
+  vanuatuHeight: 28759681
```
Comment (dustinxie) [inline genesis_testnet.yaml:37]: placeholder time Tuesday 9:30am, we can determine the specific time later

File: genesis_testnet.yaml [hunk 5]
```diff
@@ -33,6 +34,7 @@ blockchain:
   sumatraHeight: 24527521
   tsunamiHeight: 25226641
   upernavikHeight: 26984161
+  vanuatuHeight: 28761841
```
Comment (dustinxie) [inline genesis_testnet.yaml:37]: US time: Oct 28 10pm, China time: Oct 29 1pm

**Review by CoderZhi — DISMISSED** [review]

**Review by envestcc — DISMISSED** [review]

**Review by envestcc — APPROVED** [review]

**Review by guo — APPROVED** [review]

**Pull request merged**